CHANGELOG.md

```diff
@@ -1,3 +1,128 @@
+# 4.0.0
+This version includes some big changes, so we've decided to make it version 4.0!
+
+- webdriver-manager is now it's [own NPM](https://www.npmjs.com/package/webdriver-manager), so you
+can use it in your own projects. Protractor depends on it, though, so you shouldn't need to change
+anything.
+
+- Protractor has TypeScript typings! See the [example](https://github.com/angular/protractor/tree/master/exampleTypescript)
+for details on how to use TypeScript in your tests.
+
+## Breaking changes
+- ([d932ad7](https://github.com/angular/protractor/commit/d932ad7e853c0bda5d45b478a5c0271d072b6794)) 
+  chore(browser): rename protractor to browser and add a protractor namespace (#3214)
+
+  * added wrapDriver method from the browser.ts and ExpectedConditions to the protractor namespace
+  * imported selenium webdriver ActionSequence, Key, promise, Command, and CommandName to the
+  protractor namespace
+
+- Selenium Webdriver has deprecated getInnerHTML and getOuterHTML. You'll need to update your tests to
```

Comment:
Nit: on capitalization [getInnerHtml](http://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.getInnerHtml) and [getOuterHtml](http://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.getOuterHtml)